src/hooks/usePriceUsd.ts

```diff
@@ -0,0 +1,13 @@
+
+import {useEffect} from 'react'
+import {usePriceCakeBusd } from 'state/hooks'
+
+const usePriceUsd = () => {
+    const cakePriceUsd = usePriceCakeBusd()
+    useEffect(() => {
+    document.title = `PancakeSwap - $${Number(cakePriceUsd).toLocaleString(undefined, {minimumFractionDigits: 3, maximumFractionDigits: 3})}`
+    }, [cakePriceUsd])
+    return null
```

Comment:
Does eslint throw an error on this? If it _doesn't_ please remove.

Reply:
It doesn't I will remove that